lib/report-builder-factory/index.js

```diff
@@ -1,13 +1,18 @@
 'use strict';
 
-const ReportBuilder = require('./report-builder');
+const ReportBuilderJson = require('./report-builder-json');
+const ReportBuilderSqlite = require('./report-builder-sqlite');
+
 const adapters = {
     gemini: require('../test-adapter/gemini-test-adapter'),
     hermione: require('../test-adapter/hermione-test-adapter')
 };
 
 module.exports = {
-    create: (toolName, tool, pluginConfig) => {
-        return ReportBuilder.create(tool, pluginConfig, adapters[toolName]);
+    create: (toolName, tool, pluginConfig, isSqlite = false) => {
+        if (isSqlite) {
```

Comment:
We need this flag because it determines what kind of ReportBuilder we need to create -- a json or a sqlite one. Different places in code require different ReportBuilders and this flag is the only way to specify what reporter we need to return

plugin.config only specifies what kind of saveFormat we're using - the old one (only data.js) or the new (sqlite.db + data.js)